src/data/gitmojis.json

```
@@ -419,6 +419,13 @@
       "code": ":wastebasket:",
       "description": "Deprecate code that needs to be cleaned up.",
       "name": "wastebasket"
+    },
+    {
+      "emoji": "🛂",
+      "entity": "&#x1F6C2;",
+      "code": ":passport_control:",
+      "description": "Work on code related to authorization, roles and permissions.",
+      "name": "passport_control"
```

```suggestion
      "name": "passport-control"
```

Also, this is supposed to be kebab, not snake :D